lib/commands/file-movement.js

```diff
@@ -134,9 +144,14 @@ async function pushFileToSimulator (device, remotePath, base64Data) {
  *                          valid device ID.
  * @param {string} remotePath - The remote path on the device. This variable can be prefixed with
  *                              bundle id, so then the file will be uploaded to the corresponding
- *                              application container instead of the default media folder, for example
- *                              '@com.myapp.bla/RelativePathInContainer/111.png'. The '@' character at the
- *                              beginning of the argument is mandatory in such case.
+ *                              application container instead of the default media folder. Use
+ *                              @<app_bundle_id>:<optional_container_type>/<path_to_the_file_or_folder_inside_container>
+ *                              format to pull a file or a folder from an application container of the given type.
+ *                              Possible container types are 'documents' or other. Other means do not specify the container.
```

Comment:
It would be better to say: `The only supported container type is "documents". If the container type is not set explicitly for a bundle id then the default application container is going to be mounted (aka `--container` ifuse argument)`